debian/copyright

```diff
@@ -58,7 +58,7 @@ License: GPL 3
 
 Files: rednotebook/external/txt2tags.py
 Copyright: Copyright 2001, 2002, 2003, 2004, 2005, 2006, 2007, 2008 Aurelio Jargas
-License: GPL-2
+License: GPL-2+
```

Comment:
Shouldn't this stay `GPL-2` since the license text doesn't include the "or (at your option) any later version" part?